### ISSUE TITLE
Crash when mining monitor

### ISSUE THREAD

**Issue by JomerDev** [SquidDev-CC/CC-Tweaks#95]
I crashed when I tried to mine a CC advanced monitor.

I also disconnected every time I right clicked on the monitor, which I'm guessing also has something to do with it.

```
java.lang.RuntimeException: You are not attached to this Computer
    at dan200.computercraft.core.apis.PeripheralAPI$PeripheralWrapper.queueEvent(PeripheralAPI.java:221)
    at org.squiddev.cctweaks.core.network.modem.PeripheralAccess.queueEvent(PeripheralAccess.java:110)
    at dan200.computercraft.shared.peripheral.monitor.TileMonitor.func_145845_h(TileMonitor.java:250)
    at net.minecraft.world.World.func_72939_s(World.java:1939)
    at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:489)
    at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:636)
    at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
    at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
    at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
```

Tell me if you need the complete log. The mods should be the same as the ones in the last Issue I created

**Comment by SquidDev** [comment]
Sorry for taking an age to get to this. However, I can't seem to reproduce. Do you think you could send me a screenshot of your setup and maybe of the network too? I'm guessing something is disconnecting even though it shouldn't be. Possibly related to the network issues you mentioned in #92.

**Comment by JomerDev** [comment]
The setup doesn't exist anymore, but it looked something like this:
M: Monitor
C: Computer

MMMM
MMMM
MMMM
         C

with normal blocks all around.

I've had multiple problems with Tile Entities being wierd or broken on the server, so it could be caused by that.

**Comment by MJRLegends** [comment]
@SquidDev I getting the same crash on my server and the report is he mined the computer with a program running http://pastebin.com/gDhMrcgW

**Comment by MJRLegends** [comment]
@SquidDev any thought on this?

**Comment by SquidDev** [comment]
@MJRLegends Sorry, I'm currently away and can't look at this right now. I'll have a look in a couple of weeks. Do you have what code was running at the  time?

**Comment by MJRLegends** [comment]
@SquidDev no problem and nope i dont sorry as i run a few big servers and it would be hard to find the one that crashed it

**Comment by MJRLegends** [comment]
I can try and find it since i have some coords to can go off in the crash report

**Comment by SquidDev** [comment]
I'm still being able to reproduce this which is a real pain. Any other information: screenshots, which monitor on the array was being mined, etc.. would be great. Sorry :frowning:

**Comment by SquidDev** [comment]
@Nathan1852 @MJRLegends I've fixed this on the 1.8.9 branch: do either of you want the fix backported to 1.7.10? If not I'll just mark this issue as resolved. Sorry for not fixing this sooner.

**Comment by MJRLegends** [comment]
Yeah i would like the 1.7.10 back port for it

**Comment by SquidDev** [comment]
Fixed! There are a couple of things that I need to add to CCTweaks-Lua then I'll release this. This should all happen in the next few days.

**Comment by SquidDev** [comment]
This has been released in [1.7.10-1.1.0](https://github.com/SquidDev-CC/CCTweaks/releases/tag/v1.7.10-1.1.0). It should appear on [CurseForge](https://minecraft.curseforge.com/projects/cctweaks) soon.

Post something here if it doesn't actually fix it.
